creator-node/src/utils/clusterUtils.ts

```diff
@@ -20,16 +20,26 @@ class ClusterUtils {
     this._specialWorkerId = specialWorkerId
   }
 
+  /**
+   * Returns true if cluster mode is completely disabled, meaning
+   * everything runs on one process with no primary or workers.
+   */
+  isClusterDisabled() {
```

Comment:
lolol, prob we should just do `isClusterModeEnabled()` for consistency

Reply:
or change the envvar, idc - just consistency either way

Reply:
changed to isClusterModeEnabled!